lib/ttnt/internals.rb

```diff
@@ -0,0 +1,9 @@
+require 'rake'
+
+module TTNT
+  class << self
+    def root_dir
+      @root_dir ||= Rake.application.find_rakefile_location[1]
```

Comment:
What do you think about using a class variable rather than an instance variable here ? This seems more relevant.